fhir-persistence-schema/src/main/resources/add_any_resource.sql

```diff
@@ -1,5 +1,5 @@
 -------------------------------------------------------------------------------
--- (C) Copyright IBM Corp. 2016, 2019
+-- (C) Copyright IBM Corp. 2016, 2020
 --
 -- SPDX-License-Identifier: Apache-2.0
```

Comment:
only change copyright year?

Reply:
Yes, as I started with your branch, I think it's an artifact.